deployment.py

```diff
@@ -15,18 +15,20 @@
 
 from typing import Any
 
+from aws_cdk import Stack
```

Comment:
I'd prefer to keep the module namespace for a reader to quicker understand which module `Stack` and `Stage` classes come from, as there is no "natural" mapping in my mind. That is unlike `Construct` which is very generic:
```suggestion
import aws_cdk as cdk
```